graphql_compiler/tests/test_input_data.py

```diff
@@ -192,7 +192,8 @@ def name_or_alias_filter_on_interface_type():
 def output_source_and_complex_output():
     graphql_input = '''{
         Animal {
-            name @filter(op_name: "=", value: ["$wanted"]) @output(out_name: "animal_name")
+            name @filter(op_name: "=", value: ["$wanted"])
```

Comment:
Any particular reason to make this whitespace-only change? The line was under 100 chars.

Reply:
I thought we were keeping different directives on separate lines (fixed now).